docs/community.vmware.vmware_host_snmp_module.rst

```diff
@@ -238,6 +238,36 @@ Parameters
                         <div>Enable, disable, or reset the SNMP agent.</div>
                 </td>
             </tr>
+            <tr>
+                <td colspan="1">
```

Comment:
Do not generate RST documents. These will be auto-generated at the time of the release.